src/service/db/mod.rs

```diff
@@ -27,7 +27,6 @@ pub mod compact;
 pub mod dashboards;
 pub mod enrichment_table;
 pub mod file_list;
-pub mod folders;
```

Comment:
The `service/db/folder` module doesn't really do anything now that folders have been moved into their own table